### PR TITLE
update docstring doc

### PR DESCRIPTION
**Pull request by leckie-chn**

### CODE REVIEW

File: docs/en_US/Tutorial/Contributing.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ A person looking to contribute can take up an issue by claiming it as a comment/
 ## Code Styles & Naming Conventions
 * We follow [PEP8](https://www.python.org/dev/peps/pep-0008/) for Python code and naming conventions, do try to adhere to the same when making a pull request or making a change. One can also take the help of linters such as `flake8` or `pylint`
 * We also follow [NumPy Docstring Style](https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html#example-numpy) for Python Docstring Conventions. During the [documentation building](Contributing.md#documentation), we use [sphinx.ext.napoleon](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html) to generate Python API documentation from Docstring.
+* For docstrings, please refer to [numpydoc docstring guide](https://numpydoc.readthedocs.io/en/latest/format.html) and [pandas docstring guide](https://python-sprints.github.io/pandas/guide/pandas_docstring.html)
+    * For function docstring, **description**, **Parameters**, and **Returns**/**Yields** are mandatory.
```
Comment (QuanluZhang) [inline docs/en_US/Tutorial/Contributing.md]: for description, users have to write a one sentence summary and another paragraph for detailed description?

Comment (liuzhe-lz) [reply docs/en_US/Tutorial/Contributing.md]: It seems numpy and pandas docstring formats are not exactly the same. For example pandas add a line break after opening `"""`, while numpy does not.
Please declare which one has higher priority when they are in conflict.

File: docs/en_US/Tutorial/Contributing.md [hunk 2]
```diff
@@ -40,6 +40,9 @@ A person looking to contribute can take up an issue by claiming it as a comment/
 ## Code Styles & Naming Conventions
 * We follow [PEP8](https://www.python.org/dev/peps/pep-0008/) for Python code and naming conventions, do try to adhere to the same when making a pull request or making a change. One can also take the help of linters such as `flake8` or `pylint`
 * We also follow [NumPy Docstring Style](https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html#example-numpy) for Python Docstring Conventions. During the [documentation building](Contributing.md#documentation), we use [sphinx.ext.napoleon](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html) to generate Python API documentation from Docstring.
+* For docstrings, please refer to [numpydoc docstring guide](https://numpydoc.readthedocs.io/en/latest/format.html) and [pandas docstring guide](https://python-sprints.github.io/pandas/guide/pandas_docstring.html)
```
Comment (liuzhe-lz) [inline docs/en_US/Tutorial/Contributing.md]: It seems numpy and pandas docstring formats are not exactly the same. For example pandas add a line break after opening `"""`, while numpy does not.
Please declare which one has higher priority when they are in conflict.

File: docs/en_US/Tutorial/Contributing.md (hunk 1 above)
Comment (leckie-chn) [reply docs/en_US/Tutorial/Contributing.md]: This is a recommended format of **Description** Section, but not mandatory.

File: docs/en_US/Tutorial/Contributing.md (hunk 2 above)
Comment (liuzhe-lz) [reply docs/en_US/Tutorial/Contributing.md]: Btw I'm using pandas format for now.

**Pull request merged**